src/components/SimpleTable.jsx

```diff
@@ -96,7 +96,7 @@ const DataRows = ({rowData, baseStyle, columnHeaders, rowWrapper = ({renderedRow
     const id = rowData[index][0].id;
     const mapKey = id || `noId-index-${index}`;
     if (rowData[index][0].striped) {
-      baseStyle.backgroundColor = index % 2 === 0 ? 'white' : '#e2e8f4';
+      baseStyle.backgroundColor = index % 2 === 0 ? 'white' : '#F7F8F9';
```

Comment:
This globally affects the other tables that are based on simple tables. Is this expected?

Reply:
Yes, because the striped value has to get passed in the rowData for the table to be affected.